src/plots/cartesian/layout_attributes.js

```diff
@@ -899,6 +928,26 @@ module.exports = {
             'axis will be visible.'
         ].join(' ')
     },
+
+    minor: {
+        tickmode: tickmode,
+        nticks: nticks,
+        tick0: tick0,
```

Comment:
Do we actually want to support `tick0` here? Strange things could happen, which may be fine if (a) `axis.minor.tick0` inherits from `axis.tick0` (even if `axis.tick0` is set during autotick as in the case of week ticks), and (b) we test out the relevant cases and ensure we like the results.

I guess this is the main thing we want to test in jasmine, right? First the standard cases, where you provide major tick settings and various combinations of minor `nticks`, `dtick`, and `tick0` and verify that the correct minor ticks are generated.

Even without explicit `tick0` there are questions about what to do when `dtick` is not an integer multiple of `minor.dtick` - do we keep the minor tick spacing consistent, or do we start fresh from every major tick? Presumably the former, even though that means two minor ticks will straddle one major. But I could certainly see wanting this for example to put week minor ticks with month major ticks.